assets/js/vendor.js

```diff
@@ -13772,6 +13772,13 @@ return VueRouter;
 	});
 
 	$.fn.extend({
+    	/*
+		* Replaces space encoded character for serialize function
+		*/

```

Comment:
The formatting for docblocks should be adjusted slightly. https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

```php
/**
 * Replaces space encoded character for serialize function
 */
```

Personally, I would prefer a more descriptive summary.

```php
/**
 * Custom jQuery serialize wrapper.
 *
 * When WordPress 5.6 increased the jQuery version to 3.5.1, the serialize function changed. Instead of
 * sending spaces as "+", they are sent as "%20". This wrapper is for backwards compatibility.
 *
 * @since ###
 */
```

Reply:
Script is every page that needs it. The description has been updated as well.

Reply:
The formatting of the docblock is still slightly off:
![image](https://user-images.githubusercontent.com/5925563/117460008-06060380-af1a-11eb-9db7-ea61b9584de9.png)

This does not affect the code at all, and honestly is a waste of time. Hence why some repos have automated this process and used linters instead. Please see https://www.perforce.com/blog/qac/what-lint-code-and-why-linting-important

In the mean time, I just want to make sure you're aware of how specific the rules can be in some cases.